README.md

```diff
@@ -1,53 +1,109 @@
-# Wiki
+# Publish to GitHub wiki action
 
-This action takes the contents of a provided directory and synchronizes it with your
-repository's wiki. This allows, for example, pull requests for the wiki.
+📖 Deploy docs from your source tree to the GitHub wiki
 
-The goal is to expose the wiki to PRs, so that potential contributors who aren't
-collaborators can contribute to the wiki and go through a review process. Alternatively,
-one can disabled the "Restrict editing to collaborators only" setting for their wiki,
-but this allows the community to make unreviewed, potentially unwanted, changes.
+<div align="center">
 
-By default, it will use the `.github/wiki` directory.
+![](https://placekitten.com/600/400)
 
-## Before using this action
+<!-- prettier-ignore -->
+[Awesome GitHub wikis](https://github.com/MyHoneyBadger/awesome-github-wiki#readme)
+| [🆕 What's new in v1](#tbd)
+| [GitHub workflow permissions docs](https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs)
 
-You probably want to commit your wiki in its initial state before using this action.
-This is because this action will commit *all* differences between the wiki and the
-`.github/wiki` (or other specified directory) contents.
+</div>
 
-```bash
-git clone https://github.com/octocat/my-repo.wiki.git .github/wiki
-rm -rf .github/wiki/.git
-git add .github/wiki
-git commit -m "Add wiki to repo"
-```
+🌐 Works across repositories (with a [PAT]) \
+📚 Pretty interface for Markdown docs \
+⤴️ Lets you open PRs for wiki docs
```

Comment:
```suggestion
## Features

- 🌐 Works across repositories (with a [PAT])
- 📚 Pretty interface for Markdown docs
- ⤴️ Lets you open PRs for wiki docs
```
Accessibility

Reply:
🤷‍♂️ I guess...

If you're concerned about screen readers, they do actually IMO a better job _without_ all the extra markup there! I used ChromeVox (I was on a Chromebook writing this) and got the following output spoken:

Current version (just `<br>` with no list markup):

```
globe works across repositories with a pat books pretty interface for markdown docs
```

Here's what it spoke using the `## Features` thing:

```
heading 2 features bullet globe works across repositories with a pat list item list with 3 items bullet books pretty interface for markdown docs...
```

I did some searching for other product feature adverts, for all their like "30 pb of data transferred, 10 million users" lists, they use just divs and flexbox (sample size of 3). Point is that it's OK to not be semantically perfect with headings and labels for everything. I know "others are doing it" isn't a justification to ignore accessibility, but in this case, it's all text with with no hidden weird page jumps and breaks and CSS and stuff, so I think it's OK to use the emojis as the "bullet points" instead of using an actual list.

But that's me. I guess I can change it. imo it doesn't look quite as clean with the heading and the prefix dot of the actual bullet list.

Reply:
K, I'm fine with not using bullet points though. I think the heading has use, though, since it will become part of GH's generated table of contents, and also provide an element with an ID that we would be able to link to.

Reply:
🤷‍♂️ fixed in https://github.com/jcbhmr/actions-wiki/commit/158a596f1db002fdbad1e5088d92d63d2c8d4558

Seems like such a _itty-bitty_ 🤏 section though 🤷‍♂️

![image](https://user-images.githubusercontent.com/61068799/225719875-2cac51e3-5b8f-4771-a5cd-7c22b70c9ba3.png)